test/MUXControlsTestApp/master/AppBarToggleButton.xml

```diff
@@ -26,7 +26,7 @@
       FocusVisualPrimaryThickness=2,2,2,2
       FocusVisualPrimaryBrush=#FF000000
       Visibility=Visible
-      RenderSize=68,56
```

Comment:
>68,56 [](start = 17, length = 5)

Its possible this file just needs to be reverted. I don't think your change would have caused appBarTOggleButton to be changed

Reply:
I am happy to revert the file, just let me know if that is what I should do.

Reply:
please revert this, then i'll rerun

Reply:
I reverted the changes to this file, should I do that for any other files as well?